codegen-client-test/build.gradle.kts

```diff
@@ -61,31 +61,37 @@ data class ClientTest(
 
 val allCodegenTests = listOf(
     ClientTest("com.amazonaws.simple#SimpleService", "simple", dependsOn = listOf("simple.smithy")),
+    /*  TODO(AuthAlignment): Comment in once the no auth scheme resolver can be injected via client config
```

Comment:
Wouldn't test credentials still work? How does this work now?

Reply:
I was asking why we need to inject no auith scheme resolver in the first place when test credentials would work?